src/browser.ts

```diff
@@ -139,6 +170,7 @@ export function createApiCall(
   descriptor?: Descriptor
 ): GaxCall {
   if (
+    descriptor !== null &&
```

Comment:
Not sure why this change got in this PR :)  but anyway, why don't we just do `if (descriptor && decsriptor.constructor,name === '.....')`)

Reply:
When Ramya and I were working on some samples for streaming a while ago, we came across cases where the descriptor wouldn't be undefined specifically and had to cover the null case.

Done.